README.md

```diff
@@ -1 +1,82 @@
-# hello_microverse
+📗Table of Contents
+
+📖 About the Project
+🛠 Built With
+Tech Stack
+Key Features
+💻 Getting Started
+Prerequisites
+Setup
+Usage
+👥 Authors
+🤝 Contributing
+🙏 Acknowledgements
+📝 License
+
+📖 [Hello Microverse]
+
```

Comment:
- [ ] As the last code reviewer suggest 👇🏽 

> Please make sure that your Table of the content section of your readme file is linked to each section of your readme file.

![image](https://user-images.githubusercontent.com/58642949/205613004-ad8d69e9-b54d-49c5-8179-5063dbb4a645.png)

I recorded this video to help you show how to copy the [README template](https://github.com/microverseinc/readme-template) 

https://user-images.githubusercontent.com/58642949/205617959-e0f050b5-39ce-49d2-a206-7189eec2a284.mp4


- [ ] **[OPTIONAL]** Please 🙏🏽 add the [murple_logo.png](https://github.com/microverseinc/readme-template/blob/master/murple_logo.png) file to your repo to have a professional README file.